src/content/contributing/style-guide/index.md

```diff
@@ -0,0 +1,200 @@
+---
```

Comment:
We should link to this page from https://ethereum.org/en/contributing/, probably the "How to work on ethereum.org" section.